### DISCUSSION TITLE
[feature request for WIP feature] a few things about arithmetic

### DISCUSSION THREAD

**Discussion by User670** [26F-Studio/Techmino#305] [Ideas]
1. why do single-digit multiplication come after two-digit multiplication?
2. would rather different bases (base 2, base 8, base 16) have words saying what base it is, rather than use just color
3. consider bases other than 2, 8 and 16?

**Comment by MrZ626** [comment]
1. This should be a issue (fixed)
2. I'll consider it
3. not neccessary

[a comment by an automated account is not shown]
